module-new/.storybook/main.ts

```diff
@@ -1,21 +1,14 @@
 import type { StorybookConfig } from '@storybook/react-vite';
 const config: StorybookConfig = {
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-  addons: [
-    '@storybook/addon-links',
-    '@storybook/addon-essentials',
-    '@storybook/addon-interactions',
-    '@storybook/addon-a11y',
-    '@storybook/test-runner',
-    'storybook-addon-designs',
-  ],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-interactions', '@storybook/addon-a11y', '@storybook/test-runner', 'storybook-addon-designs', '@storybook/addon-mdx-gfm'],
   framework: {
     name: '@storybook/react-vite',
-    options: {},
+    options: {}
   },
   features: {},
   docs: {
-    autodocs: true,
-  },
+    autodocs: true
+  }
 };
-module.exports = config;
+module.exports = config;
```

Comment:
It looks like your auto-formatter is working against the prettier setup here, these changes should be removed